### PR TITLE
[crc32c]:For non Intel machines default behavior is changed

### PR DESCRIPTION
**Pull request by ossdev07**
A deafult software fallback is added for non Intel machines, which was
previously giving import error.

Signed-off-by: ossdev07 <ossdev@puresoftware.com>

### CODE REVIEW

**Comment by rtobar** [comment]
Hi @ossdev07 thanks for the contribution. This is obviously the core change that needs to happen, but as I mentioned in #12 changing this behaviour implies more changes than just removing the failure to import (new env variable, deprecate the old one, add info to indicate whether hw/sw method is in use). Would you be able to address those?

I also am not sure the usage of `eval` for printing is optimal, I assume it could be changed for something else.

We can ignore the Travis CI failure, it's homebrew-related and those are usually (but not alwayge) intermittent, and have nothing to do with your change.

File: _crc32c.c [hunk 1]
```diff
@@ -116,8 +116,12 @@ MOD_INIT(crc32c)
...
 
 	else {
-		PyErr_SetString(PyExc_ImportError, import_error_msg);
-		return MOD_VAL(NULL);
+#if PY_MAJOR_VERSION >= 3
+		PyRun_SimpleString("print('SSE4.2 extensions providing a crc32c hardware instruction are NOT available in your processor.  A software implementation of the same has been loaded instead.')");
+#else
+		PyRun_SimpleString("print 'SSE4.2 extensions providing a crc32c hardware instruction are NOT available in your processor.  A software implementation of the same has been loaded instead.'");
+#endif	
+		crc_fn = _crc32c_sw_slicing_by_8;
```
Comment (rhenwood-arm) [inline _crc32c.c]: You can use brackets with 'print' in python2 - so it is not clear to me that you need this test for python3?

File: _crc32c.c [hunk 2]
```diff
@@ -116,8 +116,12 @@ MOD_INIT(crc32c)
 	}
 
 	else {
-		PyErr_SetString(PyExc_ImportError, import_error_msg);
-		return MOD_VAL(NULL);
+#if PY_MAJOR_VERSION >= 3
+		PyRun_SimpleString("print('SSE4.2 extensions providing a crc32c hardware instruction are NOT available in your processor.  A software implementation of the same has been loaded instead.')");
```
Comment (rhenwood-arm) [inline _crc32c.c]: I think a more precise error message can be stated. Something like:
"crc32c hardware is either unavailable or inaccessible from this library today. A software implementation will be used instead."

**Comment by rtobar** [comment]
This PR is now superseeded by the latest changes in the `master` branch, which implement a more thorough change in the import logic. I'm thus closing this PR.

**Pull request closed without merging**